src/controllers/profile.js

```diff
@@ -0,0 +1,36 @@
+const r = require('express').Router();
+
+r.get('/', (req, res) => {
+
+
+    res.render('profile');
+});
+
+//change password
+r.post('/profile/password', (req, res) => {
+    const {password, new_password, new_passwordConfirm} = req.body;
+
+    if (new_password == new_passwordConfirm) {
+        // creating a new password
+        if (new_password !== password) {
+
+            //TODO: delete old password
+
+            //TODO: insert new password
+            const stmt = req.database
+                .prepare(`INSERT INTO users (password_hash)
```

Comment:
INSERT fügt eine neue zeile hinzu - du willst ja nur eine spalte eines bestehenden eintrags ändern. guck mal in meinem scoreboard controller wie ich den UPDATE befehl nutze.